xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -247,6 +247,15 @@ enum AVPixelFormat CDVDVideoCodecFFmpeg::GetFormat( struct AVCodecContext * avct
       {
         ctx->m_processInfo.SetVideoPixelFormat(pixFmtName ? pixFmtName : "");
         ctx->SetHardware(dec);
+
+        std::list<EINTERLACEMETHOD> deintMethods;
```

Comment:
This path is for MMALFFMPEG - the zero copy ffmpeg acceleration.
That doesn't currently have access to processInfo.
MMALCodec (full hardware decode) does do it directly.